exporter/signalfxexporter/internal/translation/converter.go

```diff
@@ -234,7 +237,7 @@ func (dpv *datapointValidator) sanitizeDimensions(dimensions []*sfxpb.Dimension)
 
 func (dpv *datapointValidator) isValidMetricName(name string) bool {
 	if len(name) > maxMetricNameLength {
-		dpv.logger.Warn("dropping datapoint",
+		dpv.logger.Debug("dropping datapoint",
```

Comment:
These changes deserve a changelog entry. Can you please submit another PR with them?

Reply:
@dmitryax I update the current changelog and added this, let me know if this does it

```
subtext: Additionally, the metric and dimension validation failures are now logged in Debug instead of Warn
```